contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -2966,14 +2966,17 @@ def testTextureParameters( self ) :
 		self.assertEqual( sphere.attributeNames(), [ "surface" ] )
 		network = sphere.readAttribute( "surface", 0 )
 
+		currentDir = os.path.dirname( __file__ )
+		currentDir = currentDir[:1].lower() + currentDir[1:] if os.name == "nt" else currentDir
+
 		self.assertEqual( network.size(), 4 )
 		self.assertEqual(
 			os.path.normpath( network.getShader( "relativeTexture" ).parameters["file"].value ),
-			os.path.join( os.path.dirname( __file__ ), "myTexture.tx" ),
+			os.path.join( currentDir, "myTexture.tx" ),
```

Comment:
I think we can just test with `normcase()` here as well, rather than monkey around with `currentDir`.

Reply:
Fixed in c8c04b33c97e8f8824da450e34730de4cfa55d34